MagaluCloud/magalu
# fix: fixing vpc flag name

## Description
Fixing the vpc id flag for the command `mgc network vpc ports list`

## Progress
Done

## How to test it
You can run the command `mgc network vpc ports list` and check for a flag with the name `vpc-id`

## Visual reference
Before
![Captura de tela de 2024-01-11 10-55-21](https://github.com/profusion/magalu/assets/110136151/9d4ae537-f0d0-409b-b6a8-f5a93b53eeb1)

After
![Captura de tela de 2024-01-11 10-54-57](https://github.com/profusion/magalu/assets/110136151/55c13262-2d85-49f3-94f0-81c1dc3253a2)
